app/models/google_calendar/event_finder.rb

```diff
@@ -1,29 +1,25 @@
 module GoogleCalendar
   class EventFinder
     GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
-    LISTING_FIELDS = 'items(id, start, end, summary, recurrence, creator, attendees(self, responseStatus))'.freeze
+    LISTING_FIELDS = 'items(id, start, end, summary, recurrence, '\
```

Comment:
https://infinum.co/the-capsized-eight/articles/multiline-strings-ruby-2-3-0-the-squiggly-heredoc